diff --git a/src/depth_anything_3/api.py b/src/depth_anything_3/api.py
index 9e09235..ff5a244 100644
--- a/src/depth_anything_3/api.py
+++ b/src/depth_anything_3/api.py
@@ -32,11 +32,9 @@ from depth_anything_3.cfg import create_object, load_config
 from depth_anything_3.registry import MODEL_REGISTRY
 from depth_anything_3.specs import Prediction
-from depth_anything_3.utils.export import export
 from depth_anything_3.utils.geometry import affine_inverse
 from depth_anything_3.utils.io.input_processor import InputProcessor
 from depth_anything_3.utils.io.output_processor import OutputProcessor
 from depth_anything_3.utils.logger import logger
-from depth_anything_3.utils.pose_align import align_poses_umeyama
 
 torch.backends.cudnn.benchmark = False
 # logger.info("CUDNN Benchmark Disabled")
@@ -350,6 +348,8 @@ class DepthAnything3(nn.Module, PyTorchModelHubMixin):
         """Align depth map to input extrinsics"""
         if extrinsics is None:
             return prediction
+        from depth_anything_3.utils.pose_align import align_poses_umeyama
+
         prediction.intrinsics = intrinsics.numpy()
         _, _, scale, aligned_extrinsics = align_poses_umeyama(
             prediction.extrinsics,
@@ -415,6 +415,8 @@ class DepthAnything3(nn.Module, PyTorchModelHubMixin):
         self, prediction: Prediction, export_format: str, export_dir: str, **kwargs
     ) -> None:
         """Export results to specified format and directory."""
+        from depth_anything_3.utils.export import export
+
         start_time = time.time()
         export(prediction, export_format, export_dir, **kwargs)
         end_time = time.time()
